server.lua

```diff
@@ -459,7 +459,24 @@ function SetNickname(user, nickname)
 	if discordId then
 		local name = nickname or ""
 		local endpoint = ("guilds/%s/members/%s"):format(Config.Guild_ID, discordId)
-		local member = DiscordRequest("PATCH", endpoint, json.encode({nick = tostring(name)}))
+		local member = DiscordRequest("PATCH", endpoint, json.encode({nick = tostring(name)}), "Reason for the API call")
+		if member.code ~= 200 then
+			print("[Badger_Perms] ERROR: Code 200 was not reached. Error Code: " .. error_codes_defined[member.code])
+		end
+	end
+end
+
+function ChangeDiscordVoice(user, voice)
+	local discordId = nil
+	for _, id in ipairs(GetPlayerIdentifiers(user)) do
+		if string.match(id, "discord:") then
+			discordId = string.gsub(id, "discord:", "")
+		end
+	end
+
+	if discordId then
+		local endpoint = ("guilds/%s/members/%s"):format(Config.Guild_ID, discordId)
+		local member = DiscordRequest("PATCH", endpoint, json.encode({channel_id = voice}), "Reason for the API call")

```

Comment:
Why do we need a reason for the API call if we're just going to pass a message like this lol?... -- Maybe the message of reason should be configurable and passed in via `ChangeDiscordVoice` method?

Reply:
Since I only use the function for one thing, I figured it would be enough to let it be configured here, but I see your point, I will push a commit to let users pass a reason via the function itself.